src/main/kotlin/edu/agh/susgame/dto/socket/ClientSocketMessage.kt

```diff
@@ -47,4 +47,27 @@ sealed class ClientSocketMessage {
         val questionId: Int,
         val answer: Int,
     ) : ClientSocketMessage()
-}
+
+    @Serializable
+    data class PlayerJoiningRequest(
```

Comment:
czy to są na pewno requesty?
ja bym wywalił ten suffix bo `PlayerJoiningRequest` brzmi jakby ktoś wysyłał prośbę o dołączenie do lobby która ma zostać zaakceptowana.

Reply:
Chociaż wsm odpowiedniki serwerowe są takimi odpowiedziami więc to się nawet jakos zgrywa ale np z perspektywy gracza trzeciego dostanie on nagle `PlayerJoiningResponse` mimo, że nikdy nie wysłał requesta.

Reply:
Więc ja bym wywalił wszystkie suffixy `request` i `response`